### ISSUE TITLE
Data sources?

### ISSUE THREAD

**Issue by cjnqt** [aliasnille/pap-api#3]
What is the data source? how reliable would you say the register is?

**Comment by tobiastornros** [comment]
I don´t think that the register is reliable right now because it seems that database was latest updated 12/17/2017 if you look at the readme.

**Comment by zenny** [comment]
@aliasnille The OTs first question is unanswered still. I am also curious to learn:

**What is the data source?**

Answer to the above is important to know to determine the reliability.

Cheers,
/z
